app/users/dashboard/projects/[id]/page.tsx

```
@@ -240,429 +223,530 @@ export default function ProjectDashboardPage() {
+                    <div
+                      className={`text-sm font-medium ${
+                        daysRemaining !== null && daysRemaining < 30
+                          ? 'text-red-600'
+                          : 'text-zinc-900 dark:text-zinc-100'
+                      }`}
+                    >
+                      {daysRemaining === null
+                        ? 'Not set'
+                        : daysRemaining > 0
+                          ? `${daysRemaining} days`
+                          : 'Overdue'}
```

ESLint

[error] Nested ternary expression should be parenthesized.

(unicorn/no-nested-ternary)